### PR TITLE
Initial draft

### PR DESCRIPTION
**Pull request by bdunavant**

### CODE REVIEW

File: content/irondb/metric-names-tags-queries.md [hunk 1]
```diff
@@ -71,20 +71,33 @@ Examples:
     
 Tags (including category, colon, and value) are limited to 256 characters for each tag-pair.  Tag-pairs exceeding that length will be truncated.
 
-Tags that contain characters outside of the acceptable set can be ingested by base64 encoding.
+=== Base 64 Encoding
```
Comment (esproul) [inline content/irondb/metric-names-tags-queries.md:74]: please use the `#` form of heading style, for consistency.

Comment (esproul) [inline content/irondb/metric-names-tags-queries.md:74]:
```suggestion
=== Base64 Encoding
```

File: content/irondb/metric-names-tags-queries.md [hunk 2]
```diff
@@ -71,20 +71,33 @@ Examples:
...
+
+Tags that contain characters outside of the acceptable set can be ingested, or searched for, by base64 encoding.
 To store a metric like:
 
     foo|ST[~(category):<value>]
     
-Where tilde `~`, parens `()`, and greater/less `<>` are outside of the character set you would encode
-the category and value separately as base64 and enclose them in `b""`.  For example:
+The tilde `~`, parens `()`, and greater/less `<>` are outside of the character set.  The category and value can be encoded separately as base64 and enclosed in `b""`.  For example:
```
Comment (esproul) [inline content/irondb/metric-names-tags-queries.md:81]:
```suggestion
The tilde `~`, parens `()`, and greater/less `<>` are outside of the acceptable character set.  The category and value can be encoded separately as base64 and enclosed in `b""`.  For example:
```

File: content/irondb/metric-names-tags-queries.md [hunk 3]
```diff
@@ -71,20 +71,33 @@ Examples:
...
     foo|ST[b"fihjYXRlZ29yeSk=":b"PHZhbHVlPg=="]
+    and(b"fihjYXRlZ29yeSk=":b"PHZhbHVlPg==")
     
 It is always safe to encode *all* incoming tags in this way, the server will decide if the name
 is safely representable without encoding and store the metric name decoded if it can.
 
-> Note that this encoding also applies to tag searches if the search uses an unsupported character
+=== Quoting
+
+For searching (but not ingestion) tags that contain characters outside of the acceptable set can also be quoted with double-quotes.  Double-quoted strings accept all printable ascii characters other than `"` and `\`, which must be escaped as `\"` and `\\` respectively.
```
Comment (esproul) [inline content/irondb/metric-names-tags-queries.md:91]:
```suggestion
For searching, but not ingestion, tags that contain characters outside of the acceptable set can also be quoted with double-quotes.  Double-quoted strings accept all printable ASCII characters other than `"` and `\`, which must be escaped as `\"` and `\\`, respectively.   
```

File: content/irondb/metric-names-tags-queries.md [hunk 4]
```diff
@@ -71,20 +71,33 @@ Examples:
...
-> Note that this encoding also applies to tag searches if the search uses an unsupported character
+=== Quoting
+
+For searching (but not ingestion) tags that contain characters outside of the acceptable set can also be quoted with double-quotes.  Double-quoted strings accept all printable ascii characters other than `"` and `\`, which must be escaped as `\"` and `\\` respectively.   
+
+To search for a metric like:
+
+    foo|ST[~(category):<value>]
+    
+The tilde `~`, parens `()`, and greater/less `<>` are outside of the character set.  The category and value can be quoted separately with `""`.  For example:
```
Comment (esproul) [inline content/irondb/metric-names-tags-queries.md:97]:
```suggestion
The tilde `~`, parens `()`, and greater/less `<>` are outside of the acceptable character set.  The category and value can be quoted separately with `""`.  For example:
```

**Review by esproul — CHANGES_REQUESTED** [review]
Various suggestions

**Review by postwait — APPROVED** [review]

**Review by esproul — APPROVED** [review]
LGTM

**Pull request merged**